configure.ac

```diff
@@ -1438,13 +1438,21 @@ if test "$use_external_signer" != "no"; then
     ;;
     *)
       AC_MSG_CHECKING([whether Boost.Process can be used])
+      TEMP_CXXFLAGS="$CXXFLAGS"
+      dnl Boost 1.78 requires the following workaround.
+      dnl See: https://github.com/boostorg/process/issues/235
+      CXXFLAGS="$CXXFLAGS -Wno-error=narrowing"
+      TEMP_CPPFLAGS="$CPPFLAGS"
+      CPPFLAGS="$CPPFLAGS $BOOST_CPPFLAGS"
```

Comment:
There are changes in 5dbed641eb118ab3f72a93f2b84dfc0220609f93 that are not part of the cherry-picked commit (532c64a7264dd3c7329e8839547837c57da7dbe8).

Reply:
Done. Those were from the merge conflict. Updated with the following diff to make the commits the same:
```diff
diff --git a/configure.ac b/configure.ac
index a2f425ce93..a37553de8e 100644
--- a/configure.ac
+++ b/configure.ac
@@ -1458,8 +1458,6 @@ if test "$use_external_signer" != "no"; then
       dnl Boost 1.78 requires the following workaround.
       dnl See: https://github.com/boostorg/process/issues/235
       CXXFLAGS="$CXXFLAGS -Wno-error=narrowing"
-      TEMP_CPPFLAGS="$CPPFLAGS"
-      CPPFLAGS="$CPPFLAGS $BOOST_CPPFLAGS"
       TEMP_LDFLAGS="$LDFLAGS"
       dnl Boost 1.73 and older require the following workaround.
       LDFLAGS="$LDFLAGS $PTHREAD_CFLAGS"
@@ -1467,7 +1465,6 @@ if test "$use_external_signer" != "no"; then
         [have_boost_process="yes"],
         [have_boost_process="no"])
       LDFLAGS="$TEMP_LDFLAGS"
-      CPPFLAGS="$TEMP_CPPFLAGS"
       CXXFLAGS="$TEMP_CXXFLAGS"
       AC_MSG_RESULT([$have_boost_process])
       if test "$have_boost_process" = "yes"; then
```

Reply:
The reason of the conflict is #24521 which has not been backported.

Reply:
Maybe it is easier to just backport @https://github.com/bitcoin/bitcoin/pull/24521 as well at this point?

Reply:
> Maybe it is easier to just backport @#24521 as well at this point?

Done.